fem/bilinearform.cpp

```diff
@@ -1585,9 +1585,10 @@ void MixedBilinearForm::Assemble(int skip_zeros)
          ftr = mesh->GetBdrFaceTransformations(i);
          if (ftr)
          {
-            trial_fes->GetFaceVDofs(ftr->ElementNo, trial_vdofs);
+            const int iface = mesh->GetBdrElementFaceIndex(i);
+            trial_fes->GetFaceVDofs(iface, trial_vdofs);
```

Comment:
I might have a code that uses this, I'll check how it affects the results.

Reply:
Have you found time to get to it? However, this is (/should be) the right behavior, where the dofs are retrieved for the associated face. It had been mixing the face and boundary element indices since the unification of the `FaceTransformations`. Before that, the code was valid, because the actual transformation was always a face transformation.

Reply:
I looked at it, but the code I was thinking about didn't go this path. Maybe it will show up in some of the autotests.

Reply:
That will imply to change the autotests then 😊 , but we will see. So ready to approve the PR to move it there?